src/Education.UnitTests/Layout/Logical/RuleInfoTranslatorTests.cs

```diff
@@ -84,7 +84,7 @@ public void GetRuleDescriptionSections_RootCauseSectionSetsCorrectTitle(RuleIssu
                         GetRuleInfo(new[] { new DescriptionSection("root_cause", null) }, issueType), null)
                     .ToList();
 
-            descriptionSections.Should().HaveCount(1);
```

Comment:
The Fluent test analyzer thinks `HaveCount` should be used except if there is only one, in which case `ContainSingle` should be used. Doesn't seem particularly consistent...